### DISCUSSION TITLE
trusted-set-attr issue

### DISCUSSION THREAD

**Discussion by ale82to** [uBlockO/uBO-Scriptlets#32] [General] [answered]
https://old.reddit.com/r/uBlockOrigin/comments/1g7k2gc/problem_with_trusted_scriplet/?

**Comment by uBlock-user** [comment, marked as answer]
What is the problem ?

That scriptlet was updated yesterday, so force update any filter list you have

`example.com##+js(tsa, selector, attr, value)`

**Reply by ale82to** [reply]
not a big deal but for a long time I m 100 /100 sure that it's syntax was ...different ... look at this real example : apkmirror.com##+js(tsa, style,pointer-events:initial,.downloadButton,loop)a bunch of my filters stopped working ...

**Reply by uBlock-user** [reply]
I change the body of scriptlet so naturally the syntax will also change/re-order. Anyways closing.
